h2o-algos/src/main/java/hex/mojo/GenericModel.java

```diff
@@ -0,0 +1,68 @@
+package hex.mojo;
+
+import hex.*;
+import hex.genmodel.MojoModel;
+import hex.genmodel.algos.kmeans.KMeansMojoModel;
+import hex.tree.isofor.ModelMetricsAnomaly;
+import water.H2O;
+import water.Key;
+import water.fvec.ByteVec;
+
+public class GenericModel extends Model<GenericModel, GenericModelParameters, GenericModelOutput> {
+    
+    private final MojoModel _mojoModel;
```

Comment:
How does this work in multinode? MojoModel is not Freezable

Reply:
Answered below. Link to the comment: https://github.com/h2oai/h2o-3/pull/3274#issuecomment-474359094